authenticate-users/test/FunctionsTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for application controllers
+ */
+class ControllersTest extends TestCase
```

Comment:
Why only these two test cases?

Reply:
Because I couldn't come up with a good way to test an actual E2E test without major refactoring and major jankiness

Reply:
Added more test cases!